deploy-board/deploy_board/webapp/host_views.py

```diff
@@ -75,13 +75,15 @@ def get(self, request, groupname, hostname):
         host_id = get_host_id(hosts)
         asg = get_asg_name(request, hosts)
 
+        show_terminate = get_show_terminate(hosts)
```

Comment:
Currently the host terminate requires env_name and stage_name for authentication. You could either pass env_name, stage_name or write another api based on group_name.